scripts/normalize_user_tags.py

```diff
@@ -90,18 +90,24 @@
 def migrate_source_tags(tags):
     Tag = apps.get_model('osf', 'Tag')
     for tag_name in tags:
-        tag = Tag.all_tags.get(name=tag_name[0], system=True)
-        tag.name = tag_name[1]
-        tag.save()
-        logger.info(tag_name[0] + ' migrated to ' + tag_name[1])
+        try:
+            tag = Tag.all_tags.get(name=tag_name[0], system=True)
+            tag.name = tag_name[1]
+            tag.save()
+            logger.info(tag_name[0] + ' migrated to ' + tag_name[1])
+        except Tag.DoesNotExist:
+            logger.info('Ca find tag with name {}'.format(tag_name[0]))
```

Comment:
I would still think there's value in creating a new tag, even if the old one doesn't exist to rename? Similar to other try/except comment, minimize the number of lines of code in the try block, the renaming and saving of the tag shouldn't be in there.   Alternatively, you can get_or_create the tag.  If it wasn't created, then you rename!